backend/src/main/java/de/backend/exception/NoSuchSpeciesException.java

```diff
@@ -0,0 +1,13 @@
+package de.backend.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class NoSuchSpeciesException extends IllegalArgumentException{
+
+        public NoSuchSpeciesException(String s) {
```

Comment:
Hier könnte man ja auch einfach "message" als Parameter schreiben oder sich diesen ganz sparen und default-mäßig sagen "Species not found" oder so